pkg/cache/application.go

```diff
@@ -100,8 +102,14 @@ func NewApplication(appID, queueName, user string, tags map[string]string, sched
 			{Name: string(events.UpdateReservation),
 				Src: []string{states.Reserving},
 				Dst: states.Reserving},
+			{Name: string(events.ResumingApplication),
+				Src: []string{states.Reserving},
+				Dst: states.Resuming},
+			{Name: string(events.AppTaskCompleted),
+				Src: []string{states.Resuming},
+				Dst: states.Resuming},
```

Comment:
We need to define the following transitions as well: 
```
{Name: string(events.ReleaseAppAllocation),
				Src: []string{states.Resuming},
				Dst: states.Resuming},
			{Name: string(events.ReleaseAppAllocationAsk),
				Src: []string{states.Resuming},
				Dst: states.Resuming},
```
Otherwise if the app is in Resuming state, this events are not processed and the placeholders are not deleted.